### PR TITLE
Technical Leads

### PR DESCRIPTION
**Pull request by tpendragon**
Adds a proposed set of requirements and responsibilities for being a
"technical lead" on a particular project.

### CODE REVIEW

File: technical_lead.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## Requirements to be Technical Lead
+
+1. Have been a member of the team for six months or hired for the purpose of
+   leading a particular project.
+1. Have the capacity to take on leadership of a project.
+
+## Responsibilities of a Technical Lead
+
+1. Have as comprehensive as possible knowledge of the technical workings of the
+   product and document them appropriately.
```
Comment (jrgriffiniii) [inline technical_lead.md]: 👍 I had thought to enquire about the responsibility for documentation, but I did not remember.  Thank you for adding that here.

File: technical_lead.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+1. Collaborate closely with the product owner to prioritize issues and
+   ensure the backlog is groomed appropriately. This work may be delegated or
+   shared.
+1. Act as tie-breaker in code review, and ensure reviews are happening for your
```
Comment (escowles) [inline technical_lead.md]: I'm not sure about the tie-breaker part of this — we've mostly been approving PRs with a single review, and I don't remember ever voting on one.  Maybe we should have a separate page about approving PRs?

Comment (tpendragon) [reply technical_lead.md]: I'm happy to remove this. Have we never had a case where there have been conflicting opinions on PRs?

Comment (escowles) [reply technical_lead.md]: We've had conflicting opinions, but I think we've always resolved those by having a discussion and arriving at consensus, not by voting.

**Comment by hackartisan** [comment]
We need to update this PR based on https://docs.google.com/document/d/1Ih0-4-FFh3XmOPSPl-TSWualBZ3gjOEcdzAfeMDBZ0o/edit

**Review by jrgriffiniii — APPROVED** [review]

**Review by escowles — APPROVED** [review]

**Pull request merged**